src/Renderer/Camera.js

```diff
@@ -77,43 +77,68 @@ Camera.prototype.setRotation = function setRotation(rotation) {
     this.camera3D.quaternion.copy(rotation);
 };
 
-const temp = new THREE.Vector3();
+let temp = new THREE.Vector3();
 const frustum = new THREE.Frustum();
-const obbViewMatrix = new THREE.Matrix4();
+const localViewMatrix = new THREE.Matrix4();
 const tempMatrix = new THREE.Matrix4();
 const tempBox3d = new THREE.Box3();
-
-function _prepareBox3AndMatrix(box3d, matrixWorld, visibilityTestingOffset) {
+const tempSphere3d = new THREE.Sphere();
+function centerVolume(volumes3D) {
+    if (volumes3D.type === 'Sphere') {
+        temp = volumes3D.center.negate();
```

Comment:
`negate()` will modify the `volumes3D` center.

What about this proposal:
```js
// Frustum intersection are not reliable when big numbers are involved,
// so shift the volume toward the origin.
// To compensate for this modification, we'll also modify the matrixWorld
// of the object.
function shiftVolumeTowardOrigin(originalVolume, tempContainer) {
    if (originalVolume.center) {
        // THREE.Sphere objects have a .center property
        temp.copy(originalVolume.center);
    } else if (originalVolume.getCenter) {
        // THREE.Box objects have a .getCenter method
        originalVolume.getCenter(temp);
    } else {
         throw new Error(`Unsupported volume object ${volumes3D}`);
    }
    // temp is -center
    temp.negate();
    // shift the volumes3D toward origin
    tempContainer.copy(originalVolume);
    tempContainer.translate(temp);
}
```

Then use it llike this:
```js
shiftVolumeTowardOrigin(box3d, tempBox3d);
```
ou

```js
shiftVolumeTowardOrigin(sphere, tempSphere3d);
```

WDYT? (warning: I didn't test my proposal so it may be completely broken :))